Grades.java

```diff
@@ -10,9 +10,54 @@
  * 2. The lowest grade and its index (something like: the student #3 got the lowest grade and it's 13).
  * 3. The average of the grades
  */
+
+package hw2arrays;
+
+import java.util.Scanner;
+
 public class Grades {
 
-    public static void main(String[] args) {
-        // enter your code here :)
-    }
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+		
+		System.out.print("Enter number of students:");
+		String outMessage;
+		byte numOfStud = in.nextByte();
+		byte[] grades = new byte[numOfStud]; 
+		
+		for (byte i = 0; i < grades.length; i++) {
+			System.out.print("Enter grade for student #" + i + ":");
```

Comment:
As a user, I will be surprised to know that the first index is "0" and not "1" :)

Reply:
Yes, fixed.